pyproject.toml

```diff
@@ -1,10 +1,9 @@
 [build-system]
-requires = ["setuptools>=51.2", "wheel>=0.36.2", "setuptools_scm[toml]>=6.0.1"]
+requires = ["setuptools>=51.2", "wheel>=0.36.2", "setuptools_scm>=6.3.1"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
 write_to = "src/decisionengine/framework/version.py"
-version_scheme = "post-release"
```

Comment:
Will removing this increase the version number?

Reply:
This will change it from `.postN` to `.devN`.  A `postN.devN` version is python version string compatible.

Reply:
A reasonable solution, Pat.  Thanks.